pkgs/by-name/in/installShellFiles/setup-hook.sh

```diff
@@ -233,3 +233,26 @@ installShellCompletion() {
     fi
     return $retval
 }
+
+# installBin <path> [...<path>]
+#
+# Install each argument to $outputBin
+installBin() {
+    local path
+    for path in "$@"; do
+        if (( "${NIX_DEBUG:-0}" >= 1 )); then
+            echo "installBin: installing $path"
+        fi
```

Comment:
As of #331560, this is `nixWarnLog "installBin: installing $path"`. Can also use the following:

* nixChattyLog
* nixDebugLog
* nixErrorLog (this is always)
* nixInfoLog
* nixNoticeLog
* nixTalkativeLog
* nixVomitLog
* nixWarnLog

Reply:
Can I use `nixInfoLog` instead?
It does not look like a warning to me.

Reply:
Please do. My intent in introducing named functions was to make these "what level" discussions more vibrant and useful.

Reply:
P.S.: can I wait for the merge into Master branch? I usually test things at Master to not deal with mass rebuilds.

Reply:
Merge happened.
- #332764